bin/cli/src/main.rs

```diff
@@ -88,6 +90,34 @@ struct JupiterSwap {
     rpc: Rpc,
 }
 
+#[derive(Args, Debug, Clone)]
+struct PerpPlaceOrder {
+    #[clap(long)]
+    account: String,
+
+    /// also pays for everything
+    #[clap(short, long)]
+    owner: String,
+
+    #[clap(long)]
+    market_name: String,
+
+    #[clap(long)]
+    side: String,
```

Comment:
just fyi, no need to change: `clap(value_enum)` is pretty nice for cases like this, see liquidator's main